spec/system/admin/facility_reservations_search_spec.rb

```diff
@@ -28,7 +28,7 @@
       login_as director
       visit facility_reservations_path(facility)
 
-      select instrument, from: "Products"
+      select instrument.name, from: "Products"
```

Comment:
This gets rid of a capybara warning that you're only supposed to target strings and integers, not objects.